### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by duncanwp**
I want to add a contribution guide to encourage contributions and set out a clear roadmap for development,

@AndrewWilliams3142  - could you please see what you think?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Firstly, thank you for considering contributing to ESEm! As a small unfunded package any contributions are welcome, small or large :-)
+
+## Questions
+If you have a general question regarding installing / using / understanding ESEm then please feel free to use our [Discussions](https://github.com/duncanwp/ESEm/discussions) page where someone will be able to help.
+
+## Issues
+If you have a problem with ESEm and it isn't working as documented or is producing unexpected errors then please check in the [Issues](https://github.com/duncanwp/ESEm/issues) page to see if anyone else has reported a similar problem. 
+Feel free to comment on associated issues so that we know who it's affecting. 
+
+If there isn't an appropriate existing issue, please create one and be as descriptive as possible about what you expected to happen and what actually happened. Please also explain what you've tried to fix it.
```
Comment (AndrewWilliams3142) [inline CONTRIBUTING.md:12]: Also a minimal working example to demonstrate the issue!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### Welcome contributions
+ - We're very keen to include more diverse or advanced sampling techniques
+ - Further wrappers for emulation/regression techniques (ideally using existing dependencies, such as Keras or scikit-learn)
+ - Improvements to documentation / code comments
+ - Other examples / notebooks
+ - Other data pre-processors
+
+### Contributions likely to be rejected
+ - Implementation of emulation techniques within ESEm
```
Comment (AndrewWilliams3142) [inline CONTRIBUTING.md:33]: I get what you mean here, but the line might get a bit blurry in some instances? Would suggest adding "from scratch" just to make it clearer. i.e., we don't want someone building a whole new GP/RF infrastructure within ESEm

**Comment by AndrewWilliams3142** [comment]
Great idea! Looks very helpful to me, just a couple of small comments. Also once the preprint is available it would be good to add something to README to the effect of: "If you use ESEm in your research then cite plz"

**Comment by duncanwp** [comment]
Excellent points, thank you very much!

**Pull request merged**